code/modules/mob/living/simple_animal/hostile/giant_spider/spiderling.dm

```diff
@@ -39,6 +39,7 @@
 	wanted_objects = list(/obj/machinery/atmospherics/unary/vent_pump)
 
 	environment_smash = 0//spiderlings cannot smash tables and windows anymore when getting stomped
+	var/list/spider_types = list(/mob/living/simple_animal/hostile/giant_spider, /mob/living/simple_animal/hostile/giant_spider/nurse, /mob/living/simple_animal/hostile/giant_spider/hunter)
```

Comment:
I thought it served to prevent the list from being created every time the code is executed, instead being created only once.